### PR TITLE
Expand the UDS/DogStatsD experiment set

### PR DESCRIPTION
**Pull request by blt**
### What does this PR do?

This commit updates the `uds_dogstatsd_to_api` to conform to the behavior in the `_nodist` variant excepting that we maintain the ratio of metrics in the `app_server` workload check. We also introduce a `_nomultivalue` variant of the baseline experiment that is intended to probe Agent's behavior when multiple values are not packed into the same metric payload.

REF SMP-664

### CODE REVIEW

File: test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+          tag_value_length_maximum: 100
+          tags_per_msg_minimum: 2
+          tags_per_msg_maximum: 50
+          multivalue_count_minimum: 2
+          multivalue_count_maximum: 32
+          multivalue_pack_probability: 0.08
+          kind_weights:
+            metric: 100
+            event: 0
+            service_check: 0
```
Comment (GeorgeHahn) [inline test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml:26]: Could you add an experiment to cover these?

**Review by GeorgeHahn — APPROVED** [review]
This looks great. I left one request inline.

File: test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml (hunk 1 above)
Comment (blt) [reply test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml:26]: Ah yes. I will amend the existing experiments to add these in.

Comment (blt) [reply test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml]: Addressed in [00c7869](https://github.com/DataDog/datadog-agent/pull/19324/commits/00c786932fbabe4f4ea3188fa31126e6f592e2f0)

File: test/regression/cases/uds_dogstatsd_to_api_nomultivalue/experiment.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+environment:
+  DD_TELEMETRY_ENABLED: true
+
+profiling_environment:
+  DD_INTERNAL_PROFILING_ENABLED: true
+  DD_INTERNAL_PROFILING_UNIX_SOCKET: /var/run/datadog/apm.socket
+  DD_INTERNAL_PROFILING_DELTA_PROFILES: true
+  DD_INTERNAL_PROFILING_ENABLE_GOROUTINE_STACKTRACES: true
+  DD_INTERNAL_PROFILING_EXTRA_TAGS: experiment:uds_dogstatsd_to_api
```
Comment (GeorgeHahn) [inline test/regression/cases/uds_dogstatsd_to_api_nomultivalue/experiment.yaml:12]: Copy paste error - this needs `_nomultivalue`

Comment (blt) [reply test/regression/cases/uds_dogstatsd_to_api_nomultivalue/experiment.yaml]: Addressed in [d74f558](https://github.com/DataDog/datadog-agent/pull/19324/commits/d74f55823c583a014edf63f66040ec0ab9cbb118)

**Pull request merged**